expertise/run.py

```diff
@@ -18,23 +17,37 @@
         submissions_dataset = SubmissionsDataset(submissions_file=Path(config['dataset']['directory']).joinpath('submissions.jsonl'))
 
     if config['model'] == 'bm25':
+        from .models import bm25
```

Comment:
any particular reason for importing here instead of at the top of the file?

Reply:
Yeah, I was reading just now that it is considered best practice to [import at the top of the file](https://stackoverflow.com/questions/1188640/good-or-bad-practice-in-python-import-in-the-middle-of-a-file). But I noticed that even if I was running BM25, the files used by ELMo were also loaded, which took time and were unnecessary. In that same thread I found [this answer](https://stackoverflow.com/a/1188693/12597574) as well. To avoid overhead is acceptable. What do you think?

Reply:
Right, I thought about that while commenting but I did not find either of the two classes very bulky (I might have missed something). Anyway, if you see a real difference in execution time by doing a lazy import, I think it is ok to do it.

Reply:
So the ELMo file is not the problem, but it seems that when I import ELMo, it also imports the AllenNLP libraries which are the ones that take time. Also, I thought it would be confusing for the user to import libraries that are unrelated to what they called. So if they use BM25, why should they see AllenNLP being loaded? haha. Maybe there's another way to structure the code that would avoid this and have the imports at the top at the same time. But I cannot think of one...

Reply:
How about doing this at the top of the script?
```
if config['model'] == 'bm25':
        from .models import bm25
if config['model'] == 'elmo':
        from .models import elmo
```

Reply:
I like that solution! I'll update the code. Thanks for suggesting it.